pkg/server/spec/cloudsqlproxy_test.go

```diff
@@ -64,7 +66,58 @@ func TestNewCloudSQLProxyNil(t *testing.T) {
 	}
 }
 
+func TestNewCloudSQLProxyContainerFromEnvVar(t *testing.T) {
+	a := &app.App{
+		EnvVars: []*app.EnvVar{
+			&app.EnvVar{Key: "DB_PROJECT", Value: "project"},
```

Comment:
What do you think about using a different value from `Instances` (line 78)? That way you could determine whether the test work as expected (overriding values or using `Instances`)

Reply:
@RomuloOliveira I think this test is not useful anymore. I've made tests for another function, can you please review [this one](https://github.com/luizalabs/teresa/pull/588/commits/c2547f8130dba9de898ebfd0e60003c26953ea85)? I may remove this test or just keep it there.